src/docs/get-started/codelab-web.md

```diff
@@ -133,14 +136,96 @@ and the web server when you want to test on other browsers.
 
 <li markdown="1">The starting app is displayed in the following DartPad.
```

Comment:
There's a missing </li> before this line.  I believe that is resulting in the errant `</ol>` in the [generated page at the end of step 2](https://sz-flutter.firebaseapp.com/docs/get-started/codelab-web#step-2-enable-sign-in-progress-tracking).

Reply:
fixed